shell/edit/auth/AuthProviderWarningBanners.vue

```diff
@@ -0,0 +1,33 @@
+<script>
+import { Banner } from '@components/Banner';
+
+export default {
+  name: 'AuthProviderWarningBanners',
+
+  components: { Banner },
+
+  props: {
+    tArgs: {
+      type:     Object,
+      required: true,
+      default:  () => { },
+    }
+  }
+};
+</script>
+
+<template>
+  <div>
+    <Banner
+      :label="t('authConfig.stateBanner.disabled', tArgs)"
+      color="warning"
+      data-testid="auth-provider-disabled-warning-banner"
+    />
+    <Banner
+      color="warning"
+      data-testid="auth-provider-admin-permissions-warning-banner"
+    >
+      <span v-clean-html="t('authConfig.associatedWarning', tArgs, true)" />
+    </Banner>
```

Comment:
What do we gain from having two separate banners in this instance? Would it be feasible to merge the warning into one?

Reply:
simply I forgot

Reply:
I think it can be styled to work well enough as a single banner

![image](https://github.com/rancher/dashboard/assets/835961/42a94cbd-f1aa-4f24-a38d-089fd9b1bc79)

But the double-banner doesn't look awful either

![image](https://github.com/rancher/dashboard/assets/835961/49362e41-d708-4c57-8e16-c72f1baf5d82)

Historically, we've advised against having multiple banners and merging content when possible. The goal is to avoid propagating this pattern in new code that we write, when possible.

Reply:
Sounds good, we can keep it as two banners at this point. It also looks like the guidance is to add a second banner:

https://github.com/rancher/dashboard/issues/10463#issuecomment-1991702458